pkg/data/egress_lists/egress_lists.go

```diff
@@ -0,0 +1,111 @@
+package egress_lists
+
+// TRANSITIONAL IMPLEMENTATION (UNSTABLE API)
+// This module currently provides very basic fetching of egress lists stored within the binary
+// in legacy probe/golden-AMI format. Most of its current logic & structs were borrowed from
+// osd-network-verifier-golden-ami/build/bin/network-validator.go with very little validation.
+// OSD-22628 will significantly change this module. Consider this internal API unstable until
+// that card is complete.
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"gopkg.in/yaml.v3"
+)
+
+//go:embed aws-classic.yaml
+var templateAWSClassic string
+
+//go:embed aws-hcp.yaml
+var templateAWSHCP string
+
+//go:embed gcp-classic.yaml
+var templateGCPClassic string
+
+// GetEgressListAsString returns two strings, the sum of which contains all of the URLs
+// within a given platformType's egress list. The first string returned contains all of
+// URLs with tlsDisabled=false, while the second string contains all URLs with tlsDisabled=true
+func GetEgressListAsString(platformType string, region string) (string, string, error) {
+	normalizedPlatformType, err := helpers.GetPlatformType(platformType)
+	if err != nil {
+		return "", "", err
+	}
+
+	var egressListYamlStr string
+	switch normalizedPlatformType {
+	case helpers.PlatformGCP:
```

Comment:
Without digging through this code too much - should we also check for the "new" platformtype name equivalents? i.e.:
```suggestion
	case helpers.PlatformGCP, helpers.PlatformGCPClassic:
```

Reply:
Thanks for the review! The `normalizedPlatformType, err := helpers.GetPlatformType(platformType)` a few lines up ensures that normalizedPlatformType will always be PlatformGCP, PlatformAWS, or PlatformHostedCluster. That being said, I'd love to talk about ripping out the old platform type values in a future PR so we can avoid this confusion entirely.